momento-sdk/src/main/java/momento/sdk/SimpleCacheClient.java

```diff
@@ -10,16 +10,7 @@
 import momento.sdk.exceptions.BadRequestException;
 import momento.sdk.exceptions.ClientSdkException;
 import momento.sdk.exceptions.NotFoundException;
-import momento.sdk.messages.CacheDeleteResponse;
-import momento.sdk.messages.CacheGetResponse;
-import momento.sdk.messages.CacheSetResponse;
-import momento.sdk.messages.CreateCacheResponse;
-import momento.sdk.messages.CreateSigningKeyResponse;
-import momento.sdk.messages.DeleteCacheResponse;
-import momento.sdk.messages.FlushCacheResponse;
-import momento.sdk.messages.ListCachesResponse;
-import momento.sdk.messages.ListSigningKeysResponse;
-import momento.sdk.messages.RevokeSigningKeyResponse;
+import momento.sdk.messages.*;
```

Comment:
nit - we tend to try to avoid wildcard imports.

Reply:
I need to change my intellij settings to avoid that.